### PR TITLE
Increase Volume Size For Shard 0 And Shard 1 in Mainnet Staging

### PR DESCRIPTION
**Pull request by jnels124**
* increase volume size for shard 0

### CODE REVIEW

**Review by jascks — APPROVED** [review]
LGTM

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by steven-sheehy — DISMISSED** [review]

File: clusters/mainnet-staging-na/mainnet-citus/helmrelease.yaml [hunk 1]
```diff
@@ -97,7 +97,7 @@ spec:
           - index: 0
             pods:
               persistentVolume:
-                size: 12000Gi
+                size: 13000Gi
```
Comment (steven-sheehy) [inline clusters/mainnet-staging-na/mainnet-citus/helmrelease.yaml:100]: We could probably increase by 2 TiB to give us some breathing room

Comment (jnels124) [reply clusters/mainnet-staging-na/mainnet-citus/helmrelease.yaml:100]: ok will change

**Review by jascks — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**